src/autocomplete/WildcardMentionItem.js

```diff
@@ -0,0 +1,198 @@
+/* @flow strict-local */
```

Comment:
PeopleAutocomplete: Stop using fake user-like objects for wildcard mentions
    
    As the comments point out, we've been using a hack where we make
    fake "user" objects for wildcard mentions @all and @everyone, so we
    can enlist UserItemRaw to show items for these in the people
    autocomplete.

nit: do disarm these @-mentions so that GitHub doesn't interpret them:
https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#mentioning-people

One convenient form is "@-all" / "@-everyone".

Reply:
- We now show "stream", not just "all" and "everyone", fixing #4325.

nit: https://github.com/zulip/zulip-mobile/blob/main/docs/style.md#fixes-format

(Fine to have it in prose here, but please also include the regular form -- makes it easier to spot when scanning the commit message)

Reply:
> nit: do disarm these @-mentions so that GitHub doesn't interpret them

Ooh, good catch. These didn't cross my mind as problematic; I guess either GitHub recognizes "all" and "everyone" in some special way, or maybe I've accidentally at-mentioned people with those strings as their usernames (oops!).

Reply:
Yeah, I was curious and so I expanded those commits where they appear in the main PR page here on GitHub, and hovered on the mentions -- they do in fact get recognized as mentions, of these organizations:
https://github.com/all
https://github.com/everyone

The latter has a logo for its avatar, and a description:
> We are Everyone, a digital communications agency based in Glasgow, Scotland.

Both of them appear to be ordinary non-special organizations, though as it happens not individuals.